kroxylicious-api/src/main/java/io/kroxylicious/proxy/config/secret/FilePassword.java

```diff
@@ -40,8 +41,11 @@ public String getProvidedPassword() {
 
     @Override
     public String toString() {
-        return "FilePassword[" +
-                "passwordFile=" + passwordFile + ']';
+        return "FilePassword["
+               +
```

Comment:
This look pretty weird. I like the idea of starting the line  by `+`, but here is not reformatting in the right way.

Reply:
Urgh, yes what is it thinking...